apps/lammps/scripts/setup/create_parallelcluster_config.sh

```diff
@@ -148,24 +150,21 @@ else
     return 1
 fi
 
+echo "[INFO] Create AWS ParallelCluster configuration file for LAMMPS"
+# Change the cluster configuration file
+PARALLELCLUSTER_CONFIG="${PARENT_PATH}/../../config/lammps-x86-64.yaml"
 
-# Clone crudini repository
-[ ! -d crudini ] && git clone https://github.com/pixelb/crudini
-
-
-# Install crudini
-cd crudini && pip3 install -e . && cd ..
+yq -i '.Region = strenv(AWS_REGION)' ${PARALLELCLUSTER_CONFIG}
+yq -i '.Image.CustomAmi = strenv(LAMMPS_AMI)' ${PARALLELCLUSTER_CONFIG}
+yq -i '.HeadNode.Ssh.KeyName = strenv(SSH_KEY_NAME)' ${PARALLELCLUSTER_CONFIG}
+yq -i '.HeadNode.Networking.SubnetId = strenv(SUBNET_ID)' ${PARALLELCLUSTER_CONFIG}
 
+#construct bucket url
+export BUCKET_URL="s3://$BUCKET_NAME_DATA"
 
-echo "[INFO] Create AWS ParallelCluster configuration file for LAMMPS"
-# Change the cluster configuration file
-PARALLELCLUSTER_CONFIG="${PARENT_PATH}/../../config/lammps-x86-64.ini"
-
-crudini --set ${PARALLELCLUSTER_CONFIG} "aws" aws_region_name "${AWS_REGION}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "vpc public" vpc_id "${VPC_ID}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "vpc public" master_subnet_id "${SUBNET_ID}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "cluster default" key_name "${SSH_KEY_NAME}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "cluster default" custom_ami "${LAMMPS_AMI}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "fsx parallel-fs" import_path "s3://${BUCKET_NAME_DATA}"
+yq -i '.SharedStorage[1].FsxLustreSettings.ImportPath = strenv(BUCKET_URL)' ${PARALLELCLUSTER_CONFIG}
```

Comment:
The cleanup script should be updated as well in order to read the new configuration file in yaml using `yq`.

Reply:
Good catch. Thank you. Done!